CONTRIBUTING.md

```diff
@@ -1,163 +1 @@
 # Contributing to the ForgeRock Web Login Framework
```

Comment:
Moving all of this to the docs site. We can now put something more standard for this doc.